recipe/meta.yaml

```diff
@@ -54,9 +53,7 @@ requirements:
     - asn1crypto >0.24.0,<2.0.0
     - cffi >=1.9,<2.0.0
     - cryptography >=3.1.0,<42.0.0
-    - oscrypto <2.0.0
     - pyOpenSSL >=16.2.0,<24.0.0
-    - pycryptodomex >=3.2,!=3.5.0,<4.0.0
```

Comment:
Please note that I removed these two dependencies for 3.4.0 as reported here: https://github.com/snowflakedb/snowflake-connector-python/issues/1781